extras/rya.pcj.fluo/pcj.fluo.app/src/main/java/org/apache/rya/indexing/pcj/fluo/app/observers/QueryResultObserver.java

```diff
@@ -54,6 +58,16 @@
     private static final VisibilityBindingSetStringConverter CONVERTER = new VisibilityBindingSetStringConverter();
 
     /**
+     * Simplifies Visibility expressions prior to exporting PCJ results.
+     */
+    private static final VisibilitySimplifier SIMPLIFIER = new VisibilitySimplifier();
```

Comment:
Is there a reason the simplify method isn't static (why are we creating this class)?